### DISCUSSION TITLE
Change root/super user password

### DISCUSSION THREAD

**Discussion by scb147** [hargata/lubelog#489] [Q&A] [answered]
I've been looking for a while, but how do I change the password for the root/super user account?  All other users have a profile where the password can be changed, but the root/super user account doesn't have this menu.

I was getting ready to generate a SHA256 hash of the password I want to use and update the userConfig.json file, but I thought there had to be an easier way.

**Comment by hargata** [comment, marked as answer]
If you still have access to the root user credentials, log in as the root user, go to Settings > uncheck Enable Authentication, the page will refresh, check it again and it will prompt you to set up new credentials.

If you no longer have access to the root user credentials, set EnableAuth to false in userConfig.json, navigate to your LubeLogger instance and re-check Enable Authentication in Settings.

It's kinda painful.

**Reply by scb147** [reply]
If I toggle Enable Authentication, does that remove all the registered users?

**Reply by hargata** [reply]
@scb147 nope, they will remain but for the few seconds where authentication was disabled anyone can navigate to your lubelogger instance without having to log in. The alternative to this is what you've already mentioned, generate a SHA256 password and replace it in userConfig.json

**Reply by scb147** [reply]
Thanks, I appreciate the timely responses!
